GitUI/GitUIExtensions.cs

```diff
@@ -53,7 +53,7 @@ public static class GitUIExtensions
 
             openWithDiffTool ??= OpenWithDiffTool;
 
-            if (item.Item.IsNew || firstId is null || FileHelper.IsImage(item.Item.Name))
+            if (item.Item.IsNew || firstId is null || (FileHelper.IsImage(item.Item.Name) && !item.Item.IsDeleted))
```

Comment:
Switching evaluation order to slightly improve efficiency...
```suggestion
            if (item.Item.IsNew || firstId is null || !item.Item.IsDeleted && (FileHelper.IsImage(item.Item.Name)))
```

Will change and merge